rel/overlay/etc/default.ini

```diff
@@ -278,6 +278,12 @@ bind_address = 127.0.0.1
 ; shard_cache_size = 25000
 ; shards_db = _dbs
 ; sync_concurrency = 10
+;
+; When enabled, internal replicator will replicate purge requests between shard
+; copies. It may be helpful to disable it temporarily when doing rolling node
+; upgrades from CouchDB versions 2.x to 3.x since older 2.x node do not have
```

Comment:
Hi @nickva, 
Just a detail, Clustered Purge was introduced in 2.3.0 version. So this is valid for rolling ugrades from pre 2.3.0 versions.
Thanks!

Reply:
Good point. I'll update it